kitchen.dokken.yml

```diff
@@ -68,8 +65,3 @@ platforms:
       pid_one_command: /bin/systemd
       intermediate_instructions:
         - RUN zypper --non-interactive install aaa_base perl-Getopt-Long-Descriptive which net-tools
```

Comment:
which and net-tools shouldn't be necessary here anymore https://github.com/test-kitchen/dokken-images/blob/master/fedora-latest/Dockerfile#L31

Reply:
You added this in a previous commit 😉

Reply:
Well good job me